spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -474,61 +391,335 @@ definitions:
+              0 then all correction points are valid.
+        - bitmask:
+            type: u64
+            desc: |
+              Specifies the availability of correction data at the
+              correction points in the array if bitmask_valid is true.
+
+              If a specific bit is enabled (set to 1), the correction is
+              available. Only the first rows * cols bits are used, the remainder
+              are set to 0.
+
+              Starting with the northwest corner of the array (top left on a
+              north oriented map) the correction points are enumerated with row
+              precedence - first row west to east, second row west to east,
+              until last row west to east - ending with the southeast corner of
+              the array.
+
+              See GNSS-SSR-CorrectionPoints field numberOfStepsLatitude.
+
+ - MSG_SSR_ORBIT_CLOCK_DEP_A:
+    id: 0x05DC
+    short_desc: Deprecated
+    replaced_by:
+      - MSG_SSR_ORBIT_CLOCK
+    fields:
+        - time:
+            type: GPSTimeSec
+            desc: GNSS reference time of the correction
+        - sid:
+            type: GnssSignal
+            desc: GNSS signal identifier (16 bit)
+        - update_interval:
+            type: u8
+            desc: |
+                Update interval between consecutive corrections. Encoded
+                following RTCM DF391 specification.
+        - iod_ssr:
+            type: u8
+            desc: |
+                IOD of the SSR correction. A change of Issue Of Data
+                SSR is used to indicate a change in the SSR
+                generating configuration
+        - iod:
+            type: u8
+            desc: Issue of broadcast ephemeris data
+        - radial:
+            type: s32
+            units: 0.1 mm
+            desc: Orbit radial delta correction
+        - along:
+            type: s32
+            units: 0.4 mm
+            desc: Orbit along delta correction
+        - cross:
+            type: s32
+            units: 0.4 mm
+            desc: Orbit along delta correction
+        - dot_radial:
+            type: s32
+            units: 0.001 mm/s
+            desc: Velocity of orbit radial delta correction
+        - dot_along:
+            type: s32
+            units: 0.004 mm/s
+            desc: Velocity of orbit along delta correction
+        - dot_cross:
+            type: s32
+            units: 0.004 mm/s
+            desc: Velocity of orbit cross delta correction
+        - c0:
+            type: s32
+            units: 0.1 mm
+            desc: C0 polynomial coefficient for correction of broadcast satellite clock
+        - c1:
+            type: s32
+            units: 0.001 mm/s
+            desc: C1 polynomial coefficient for correction of broadcast satellite clock
+        - c2:
+            type: s32
+            units: 0.00002 mm/s^-2
+            desc: C2 polynomial coefficient for correction of broadcast satellite clock
+
+ - STECHeaderDep:
```

Comment:
It might be pessimistic, but we should probably call this `DepA` just in case so we don't have to rename it if there's another revision to the header

Reply:
good idea